### PR TITLE
Change Hotspot Description; OncoKB updates

### PR DESCRIPTION
**Pull request by zhx828**

### CODE REVIEW

**Comment by zhx828** [comment]
This's related to #1119

**Comment by zhx828** [comment]
@onursumer  could you do a quick review and merge it?

File: portal/src/main/webapp/js/src/cbio-util.js [hunk 1]
```diff
@@ -478,6 +478,16 @@ cbio.util = (function() {
         return target;
     }
     
+    //Get hotspot description. TODO: add type as parameter for different source of hotspot sources.
```
Comment (onursumer) [inline portal/src/main/webapp/js/src/cbio-util.js]: instead of defining a utility function with a parameter, we can also define a template

**Comment by onursumer** [comment]
added a minor comment, other than that looks good to me.

**Pull request merged**